mandatory/4-tax.js

```diff
@@ -17,7 +19,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+    return "£" + parseFloat(calculateSalesTax(price)).toFixed(2);
```

Comment:
Good coding there. Use of parseFloat to avoid TypeError when applying the "toFixed" method